docs/reference/installation.rst

```diff
@@ -7,6 +7,7 @@ Installation
 
     composer require sonata-project/news-bundle sonata-project/doctrine-orm-admin-bundle
 
+If you encounter a problem with egeloen/ckeditor-bundle during installation, review the documentation here: https://sonata-project.org/bundles/formatter/master/doc/reference/installation.html
```

Comment:
I would rather see it as, if you are using Symfony4 first follow formatter installation docs, @sonata-project/contributors wdyt?

Reply:
@kunicmarko20  actually I'm finding some different issues with the documentation that negates this change. Reject this PR and when I finalize my installation I'll review where the documentation should be adjusted to make it smoother. I started installing News right away but didn't have Formatter or Media installed, which were pre-requisites but they didn't get installed and that should be called out, but need to review my steps to make sure I didn't overlook something. Thanks for the quick feedback.

Reply:
@OskarStark the project I was working on went in a different direction and I wasn't able to return to this to verify. We can close out this conversation.